src/apps/api/tests/test_competitions.py

```diff
@@ -183,56 +188,33 @@ def test_get_competition_leaderboard_as_json(self):
         content = json.loads(response.content)
 
         self.response_titles = set()
-        self.response_users = set()
         for key in content.keys():
+            response_users = set()
             title, id = key.rsplit("(", 1)
             self.response_titles.add(title)
             for user in content[key].keys():
-                self.response_users.add(user)
-        leaderboard_titles = set(self.leaderboard_ids_to_titles.values())
-        assert leaderboard_titles == self.response_titles
-        assert self.usernames == self.response_users
-
-    def test_get_competition_leaderboard_by_title_as_json(self):
-        # Makes sure the query returns a json that had a matching leaderboard title
-        leaderboard_choice = random.choice(list(self.leaderboard_ids_to_titles.values()))
-        url = reverse('competition-results', kwargs={"pk": self.comp.id})[0:-1] + f'.json?title={leaderboard_choice}'
-        response = self.client.get(url, HTTP_ACCEPT='application/json')
-        self.assertEqual(response.status_code, 200)
-        content = json.loads(response.content)
+                response_users.add(user)
+            print(f'{key} : {self.usernames} == {response_users}')
```

Comment:
pull prints from tests? or think it's handy to leave this guy in? may be worth leaving it in, but we typically don't